webtau-core/src/main/java/com/twosigma/webtau/reporter/TestListeners.java

```diff
@@ -2,43 +2,68 @@
 
 import com.twosigma.webtau.utils.ServiceLoaderUtils;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
 
 public class TestListeners {
-    private static List<TestListener> listeners = ServiceLoaderUtils.load(TestListener.class);
+    private static List<TestListener> discoveredListeners = ServiceLoaderUtils.load(TestListener.class);
```

Comment:
Should these three `private static` variables be a) final and b) in caps?

Reply:
final yes, changing. 
don't want CAPS, as they are not actually constants per se.